server/.env

```diff
@@ -1,2 +1,3 @@
 TEAM_NAME="Shums, Kim, Kevin, Charlie"
-FLASK_ENV="development"
\ No newline at end of file
+FLASK_ENV="development"
+flask_app.config['SECRET_KEY'] = 'some-secret-string'
```

Comment:
This should be:
SECRET_KEY='some-secret-string'

and in the code you would write:
flask_app.config['SECRET_KEY'] = config.SECRET_KEY

and in your config file, need to set SECRET_KEY to os.environ['SECRET_KEY']